studio/app/common/dataclass/polar.py

```diff
@@ -33,14 +32,23 @@ def output_path(self) -> OutputPath:
 
     def save_plot(self, output_dir):
         for i in range(len(self.data)):
-            fig = px.line_polar(
-                r=self.data[i],
-                theta=self.columns,
-                direction="counterclockwise",
-                start_angle=0,
-                line_close=True,
+            theta = linspace(0, 2 * pi, len(self.columns))  # Convert theta to radians
+            plt.figure()
+            ax = plt.subplot(111, polar=True)
+            ax.plot(theta, self.data[i], color="#636EFA")
+            ax.plot(
+                [theta[-1], theta[0]],
+                [self.data[i][-1], self.data[i][0]],
+                color="#FFA15A",
+                linestyle="-",
+                linewidth=2,
             )
-            plot_file = join_filepath([output_dir, f"{self.file_name}_{i}.png"])
-            pio.write_image(fig, plot_file)
-
-            save_thumbnail(plot_file)
```

Comment:
Memo:
To reduce source code differences with the folk destination repository (optinist-for-server, etc.), this code (call save_thumbnail) should remain as it was before the modification.
*Or use a flag or some other method to avoid code differences in each {plots}.py.